### ISSUE TITLE
Calculating FP

### ISSUE THREAD

**Issue by MakcuMuM** [dsiekiera/foe-helfer-extension#798]
[http://prntscr.com/rxxy5z](url) not correct calculate FP. Arka bonus is 85%

**Comment by andrgin** [comment]
You are right. FP of the bar are not counted. This will be fixed in the next update

**Comment by thevogonpoet** [comment]
I don't know if this extra info will help, but there seems to be a related (?) display bug with the FP required bar: I opened my arc and go back one level (I was just checking who contributed), I got the "You finished this level already" message, but the bar at the top says 4935/4950.

Then I go to current level, and it calculates ok, but its showing the amount of FP required from my previous level, not the current level.

So somewhere the amount of FP required is getting confused. No idea if its a bug with the helper or with FoE tho.


![fp bug](https://user-images.githubusercontent.com/63581405/79105082-39ef1380-7d70-11ea-9e75-ce4249461036.png)
